proposed/security-reporting-process-meta.md

```diff
@@ -45,22 +45,22 @@ for how security vulnerabilities go from discovery to fixing to public
 disclosure. Inspiration could be drawn from this list of security disclosure
 processes in various PHP and non-PHP projects:
 
-* http://symfony.com/security
-* http://framework.zend.com/security/
-* http://www.yiiframework.com/security/
+* https://symfony.com/security
+* https://framework.zend.com/security
+* https://www.yiiframework.com/security
 * https://www.drupal.org/security
-* http://codex.wordpress.org/FAQ_Security
-* http://www.sugarcrm.com/page/sugarcrm-security-policy/en
-* http://typo3.org/teams/security/
+* https://codex.wordpress.org/FAQ_Security
+* https://www.sugarcrm.com/legal/security-policy
+* https://typo3.org/teams/security/
 * http://cakephp.org/development
```

Comment:
I have not changed this one as it does not seem correct to me; the page is not related to security.

Reply:
The security page is available at https://book.cakephp.org/3.0/en/contributing/tickets.html#reporting-security-issues

Reply:
updated, thanks